### ISSUE TITLE
SeqMonk hangs on OSX when drawing any reports

### ISSUE THREAD

**Issue by s-andrews** [s-andrews/SeqMonk#29]
We've had a couple of reports, and one local confirmation that on some OSX systems SeqMonk will completely hang (spinning beachball requiring force quit) on some OSX systems.  It affects all reports but other dialogs appear to be OK.

**Comment by s-andrews** [comment]
I can replicate this on my mac if I use the latest Java release (v1.8.0_131-b11).

The issue does not occur using v1.8.0_111 so this appears to be a very recently introduced bug in the core JRE.

**Comment by s-andrews** [comment]
For all the use it is I've got a crash dump from OSX when I kill the hung process.
[seqmonk_crash_trace.txt](https://github.com/s-andrews/SeqMonk/files/1076928/seqmonk_crash_trace.txt)

**Comment by s-andrews** [comment]
I've got a work-round, but this is a wierd one.

I found that I could stop the crash by not calling:

```
super(application);
```

..in the ReportTableDialog constructor.  You could call super() without passing the parent in, but calling the parent messed it up for some reason.

Since this looks like a bug in the JRE I guessed that it might be a timing problem when switching windows and getting the properties of the parent.  The calling code did this:

```
	public void progressComplete(String command, Object result) {
		if (result != null) {
			new ReportTableDialog(application,report,(TableModel)result);
		}
		setVisible(false);
		dispose();
	}
```

So on a guess that it was the quick creation of a new JDialog followed by hiding and disposing of the progress box which might be triggering it I changed it to:

```
	public void progressComplete(String command, Object result) {
		setVisible(false);
		if (result != null) {
			new ReportTableDialog(application,report,(TableModel)result);
		}
		dispose();
	}
```

..and voila - no more hang.

This absolutely shouldn't make any difference so this has to be a bug, but at least we've got a work round.

**Comment by s-andrews** [comment]
I had confirmation that this was resolved on the original machine and we've now fixed this in the release as of 1.38.2.
